frontend/src/metabase/setup/tests/setup.tsx

```diff
@@ -0,0 +1,122 @@
+import { waitFor, within } from "@testing-library/react";
```

Comment:
This is pretty much all stuff extracted from the old test file, and adjusted to allow passing the enterprise plugin flag and the token features